doc/modules/manifold.rst

```diff
@@ -502,6 +502,7 @@ three parameters that control the optimization of t-SNE:
 * early exaggeration factor
 * learning rate
 * maximum number of iterations
+* angle
```

Comment:
Actually, `angle` does not really influence the optimization. It controls the approximation of the gradient. I would remove it completely and explain it in another paragraph.